testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -521,6 +529,16 @@ def test_timeseries_atomgroup_asel_mutex(self, reader):
         atoms = mda.Universe(reader.filename).select_atoms("index 1")
         with pytest.raises(ValueError, match="Cannot provide both"):
             timeseries = reader.timeseries(atomgroup=atoms, asel=atoms, order='fac')
+          
+    def test_pathlib_input_base(self, reader):     
+        if isinstance(reader, MemoryReader):
+            if isinstance(reader, MemoryReader):
+                skip_reason = "MemoryReader"
+            pytest.skip(f"Skipping test for Pathlib input with reason: {skip_reason}")
+        path = Path(reader.filename)
```

Comment:
Here adding an additional Line:
`path = path.as_posix())`

Fixes the following Failed Pytest in 4 cases:
`AttributeError: 'PosixPath' object has no attribute 'encode'`